README.md

```diff
@@ -266,7 +266,7 @@ Then copy the `result/bin/nix-installer` to the machine you wish to run it on.
 You can also add `nix-installer` to a system without Nix via `cargo`:
 
 ```bash
-RUSTFLAGS="--cfg tokio_unstable" cargo install nix-installer
+RUSTFLAGS="--cfg tokio_unstable" cargo install --git https://github.com/DeterminateSystems/nix-installer
```

Comment:
I'm not so sure we want this to be the "default" recommendation...? Maybe add a note like "if you want to try the most recent, not-yet-released changes, you can also install directly from git with: ...."

Reply:
Oh, maybe. We should do that with the nix flake ref too then